services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -68,7 +68,7 @@ def reset
     admin_teacher = User.create!(
       name: 'Toni Morrison',
       role: User::ADMIN,
-      email: teacher_email,
+      email: @teacher_email,
```

Comment:
This change shouldn't be necessary because we have an `attr_accessor` declared for `:teacher_email`.  That should make both the old and new versions of this line the same.  If you're making this change to fix some sort of issue that you were observing in testing, this may be worth an additional look.